src/adaptors/merkl/merkl-additional-reward.js

```diff
@@ -0,0 +1,68 @@
+const { networks } = require('./config');
+const { getData } = require('../utils');
+
+exports.addMerklRewardApy = async (
+  pools,
+  protocolId,
+  poolAddressGetter,
+) => {
+  try {
+    let merklPools = [];
+    let pageI = 0;
+
+    while(true) {
+      let data;
+      try {
+        data = await getData(`https://api.merkl.xyz/v4/opportunities?mainProtocolId=${protocolId}&status=LIVE&items=100&page=${pageI}`);
+      } catch (err) {
+        console.log(`failed to fetch Merkl data for ${protocolId}: ${err}`);
+        break;
+      }
+
+      if (data.length === 0) {
+        break;
+      }
+
+      merklPools.push(...data);
+      pageI++;
+    }
+
+
+    const merklPoolsMap = Object.fromEntries(Object.keys(networks).map(id => [networks[id], {}]));
+    merklPools.forEach(pool => {
+      if (!networks[pool.chainId]) {
+        return;
+      }
+
+      merklPoolsMap[networks[pool.chainId]][pool.identifier.toLowerCase()] = {
+        apyReward: pool.apr,
```

Comment:
pls confirm `pool.apr` is incentive apr only

Reply:
Yes, that's correct